src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.awt.*;
+
+public abstract class Figure {
+    String color;
```

Comment:
Consider adding a method to set the color of the figure, such as `setColor(Color color)`, to allow subclasses to set their color.